The `shardingsphere-pipeline-test` compile error

```
[INFO] Running org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest
[WARN ] 2022-03-01 10:38:47.380 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:38:54.049 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:38:58.337 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:39:01.633 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500
[WARN ] 2022-03-01 10:39:04.225 [Curator-ConnectionStateManager-0] o.a.c.f.state.ConnectionStateManager - Session timeout has elapsed while SUSPENDED. Injecting a session expiration. Elapsed ms: 20003. Adjusted session timeout ms: 20000
[WARN ] 2022-03-01 10:39:04.225 [Curator-ConnectionStateManager-0] o.a.c.f.state.ConnectionStateManager - Session timeout has elapsed while SUSPENDED. Injecting a session expiration. Elapsed ms: 20003. Adjusted session timeout ms: 20000
[WARN ] 2022-03-01 10:39:04.227 [main-EventThread] org.apache.curator.ConnectionState - Session expired event received
[WARN ] 2022-03-01 10:39:04.227 [main-EventThread] org.apache.curator.ConnectionState - Session expired event received
[WARN ] 2022-03-01 10:39:04.942 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:39:20.924 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:39:26.458 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (6000). Pinning to 3000
[WARN ] 2022-03-01 10:39:33.101 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:39:41.941 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:40:02.954 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500
[WARN ] 2022-03-01 10:40:09.594 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (6000). Pinning to 3000
[WARN ] 2022-03-01 10:40:26.183 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500
[WARN ] 2022-03-01 10:40:28.396 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:40:48.315 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:41:23.709 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500
[WARN ] 2022-03-01 10:41:39.195 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:41:44.716 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500







[WARN ] 2022-03-01 10:42:04.606 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:42:10.140 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (5000). Pinning to 3000
[WARN ] 2022-03-01 10:42:22.307 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:42:28.950 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:42:43.343 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500
[WARN ] 2022-03-01 10:43:04.368 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:43:09.896 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3000). Pinning to 1500




[ERROR] 2022-03-01 10:43:37.544 [main] o.a.s.d.p.c.a.i.RuleAlteredJobAPIImpl - Scaling job: 1, table: t_order data consistency check failed, recordsContentMatched: false, recordsCountMatched: false
[ERROR] 2022-03-01 10:43:37.544 [main] o.a.s.d.p.c.a.i.RuleAlteredJobAPIImpl - Scaling job: 1, table: t_order data consistency check failed, recordsContentMatched: false, recordsCountMatched: true
[WARN ] 2022-03-01 10:43:41.969 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:43:45.297 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:43:50.825 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:43:51.931 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2000). Pinning to 1500
[WARN ] 2022-03-01 10:43:57.466 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (2500). Pinning to 1500
[WARN ] 2022-03-01 10:44:06.320 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:44:11.846 [ShardingSphere-scaling-job-persist-0] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (3500). Pinning to 1500
[WARN ] 2022-03-01 10:44:18.480 [main] o.a.c.retry.ExponentialBackoffRetry - Sleep extension too large (6000). Pinning to 3000
[ERROR] Tests run: 10, Failures: 3, Errors: 5, Skipped: 0, Time elapsed: 337.669 s <<< FAILURE! - in org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest
[ERROR] assertResetTargetTable(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 45.66 s  <<< ERROR!
org.apache.shardingsphere.data.pipeline.core.exception.PipelineJobNotFoundException: Can not find scaling job 93363989091646
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertResetTargetTable(RuleAlteredJobAPIImplTest.java:189)

[ERROR] assertDataConsistencyCheck(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 43.123 s  <<< ERROR!
org.apache.shardingsphere.data.pipeline.core.exception.PipelineJobNotFoundException: Can not find scaling job 93409650305884
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertDataConsistencyCheck(RuleAlteredJobAPIImplTest.java:142)

[ERROR] assertDataConsistencyCheckWithAlgorithm(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 39.831 s  <<< ERROR!
org.apache.shardingsphere.data.pipeline.core.exception.PipelineJobNotFoundException: Can not find scaling job 93452773729398
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertDataConsistencyCheckWithAlgorithm(RuleAlteredJobAPIImplTest.java:153)

[ERROR] assertGetProgress(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 38.712 s  <<< ERROR!
org.apache.shardingsphere.data.pipeline.core.exception.PipelineJobNotFoundException: Can not find scaling job 93492602376675
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertGetProgress(RuleAlteredJobAPIImplTest.java:113)

[ERROR] assertRemove(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 42.006 s  <<< FAILURE!
java.lang.AssertionError
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertRemove(RuleAlteredJobAPIImplTest.java:104)

[ERROR] assertStartAndList(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 42.058 s  <<< FAILURE!
java.lang.AssertionError
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.getNonNullJobInfo(RuleAlteredJobAPIImplTest.java:85)
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertStartAndList(RuleAlteredJobAPIImplTest.java:71)

[ERROR] assertIsDataConsistencyCheckNeeded(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 42.023 s  <<< ERROR!
org.apache.shardingsphere.data.pipeline.core.exception.PipelineJobNotFoundException: Can not find scaling job 93615374312963
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertIsDataConsistencyCheckNeeded(RuleAlteredJobAPIImplTest.java:135)

[ERROR] assertStartOrStopById(org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest)  Time elapsed: 44.254 s  <<< FAILURE!
java.lang.AssertionError
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.getNonNullJobInfo(RuleAlteredJobAPIImplTest.java:85)
        at org.apache.shardingsphere.data.pipeline.api.impl.RuleAlteredJobAPIImplTest.assertStartOrStopById(RuleAlteredJobAPIImplTest.java:93)

[INFO] Running org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeEntryTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeEntryTest
[INFO] Running org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeLineTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeLineTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   RuleAlteredJobAPIImplTest.assertRemove:104
[ERROR]   RuleAlteredJobAPIImplTest.assertStartAndList:71->getNonNullJobInfo:85
[ERROR]   RuleAlteredJobAPIImplTest.assertStartOrStopById:93->getNonNullJobInfo:85
[ERROR] Errors: 
[ERROR]   RuleAlteredJobAPIImplTest.assertDataConsistencyCheck:142 » PipelineJobNotFound
[ERROR]   RuleAlteredJobAPIImplTest.assertDataConsistencyCheckWithAlgorithm:153 » PipelineJobNotFound
[ERROR]   RuleAlteredJobAPIImplTest.assertGetProgress:113 » PipelineJobNotFound Can not ...
[ERROR]   RuleAlteredJobAPIImplTest.assertIsDataConsistencyCheckNeeded:135 » PipelineJobNotFound
[ERROR]   RuleAlteredJobAPIImplTest.assertResetTargetTable:189 » PipelineJobNotFound Can...
[INFO] 

```

--- a/shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/api/impl/GovernanceRepositoryAPIImplTest.java
+++ b/shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/api/impl/GovernanceRepositoryAPIImplTest.java
@@ -40,7 +40,6 @@ import org.apache.shardingsphere.data.pipeline.scenario.rulealtered.RuleAlteredJ
 import org.apache.shardingsphere.infra.yaml.engine.YamlEngine;
 import org.apache.shardingsphere.mode.repository.cluster.listener.DataChangedEvent;
 import org.apache.shardingsphere.mode.repository.cluster.listener.DataChangedEvent.Type;
-import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
@@ -130,11 +129,6 @@ public final class GovernanceRepositoryAPIImplTest {
         ThreadUtil.sleep(1);
         JobProgress jobProgress = governanceRepositoryAPI.getJobProgress(jobContext.getJobId(), jobContext.getShardingItem());
         assertThat(jobProgress.getStatus(), is(JobStatus.FINISHED));
-    }
-    
-    @AfterClass
-    public static void afterClass() {
-        EmbedTestingServer.close();
     }
     
     private RuleAlteredJobContext mockJobContext() {

--- a/shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/fixture/EmbedTestingServer.java
+++ b/shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/fixture/EmbedTestingServer.java
@@ -136,14 +136,4 @@ public final class EmbedTestingServer {
     public static String getConnectionString() {
         return "localhost:" + PORT;
     }
-    
-    /**
-     * Close embed zookeeper server.
-     */
-    public static void close() {
-        try {
-            testingServer.close();
-        } catch (final IOException ignore) {
-        }
-    }
 }

